opsdroid/connector/matrix/connector.py

```diff
@@ -378,6 +378,25 @@ def get_roomname(self, room):
 
         return room
 
+    async def joined_rooms(self, encrypted_only=False):
+        """Returns a list of room_id of all joined rooms"""
+
+        room_ids = await self.connection.joined_rooms()
+
+        if isinstance(room_ids, nio.responses.JoinedRoomsError):
+            _LOGGER.error(
+                f"Error while getting list of joined rooms from room state: {room_ids.message} (status code {room_ids.status_code})"
+            )
+            return
+        else:
+            if encrypted_only == True:
+                room_ids = room_ids.rooms
```

Comment:
Otherwise room_ids isn't defined if `encrypted_only` is False.

```suggestion
            room_ids = room_ids.rooms
            if encrypted_only == True:
```

Reply:
Yes, my bad.